Gemfile

```diff
@@ -48,11 +48,13 @@ gem "devise"
 gem "bootsnap", ">= 1.1.0", require: false
 
 gem "sidekiq"
-
+gem "database_cleaner", group: :test
```

Comment:
A common practice when testing against the DB is to run each test in a transaction, and rollback the transaction after the test is finished. Its cleaner and faster than wiping every time. https://purelyfunctional.tv/mini-guide/clojure-database-test-faster/

Rspec supports this with some configuration: https://relishapp.com/rspec/rspec-rails/docs/transactions. You might need to require the `rails_helper` in your spec to make use of it.